AnomalyDetection/RandomCutForest/src/main/java/com/amazonaws/services/kinesisanalytics/DataSerializationSchema.java

```diff
@@ -0,0 +1,38 @@
+package com.amazonaws.services.kinesisanalytics;
```

Comment:
Missing copyright headers in all files

Reply:
Included copyright headers in all files in this PR